src/Sulu/Bundle/AdminBundle/Resources/js/views/Datagrid/Datagrid.js

```diff
@@ -99,6 +156,14 @@ class Datagrid extends React.Component<ViewProps> {
         router.navigate(addRoute, {locale: this.locale.get(), parentId: rowId});
     };
 
+    handleDelete = () => {
```

Comment:
Maybe also call this method `requestSelectionDelete`?

Reply:
should not all event handles be prefix with `handle`? 

there is also `handleItemAdd`,  and `handleEditClick` here.